book/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from book        import views
+
+urlpatterns = [
+    path('', views.BookListView.as_view()),
+    path('/detail/<int:book_id>', views.BookDetailView.as_view()),
```

Comment:
```suggestion
    path('/<int:book_id>', views.BookDetailView.as_view()),
```
RESTful API 규칙에서 특정 리소스 뒤에 `id`가 오면 그 리소스의 상세 정보임을 뜻합니다 그래서 `detail`이라는 단어는 주로 사용하지는 않습니다. 꼭 수정하시지는 않으셔도 됩니다. 일반적인 경우가 아니라는 것을 전달드립니다 :)

Reply:
빛소헌님 감사해유~
그렇다면 
`localhost:8000/book'   + `/1' 와 같이 조합하고 아래와 같이 간단하게 작성하는건가유?
(예. `localhost:8000/book/<int:book_id>`)

Reply:
@hyeseong-dev 네 맞습니담!